contracts/libraries/ProtocolGovernance.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7; 
+
+
+contract ProtocolGovernance {
```

Comment:
Looks like the intended purpose of this contract is for access control. Why not use OZ ownable?